### DISCUSSION TITLE
is there a way to render the html map  or folium maps in flet

### DISCUSSION THREAD

**Discussion by bobwatcherx** [flet-dev/flet#1008] [Q&A]
i want to display map in this flet .  and one of them with folium .  by generating html files from folium .  is there any way to render html file as widget from flet or something else to add map inside flet app . thanks

**Comment by fanxing2022** [comment]
i also want to dispaly map service  use folium by felt. thanks

**Comment by kukuk89** [comment]
I created folium maps on a flask backend with input parameter from a request from flet. Then I useed flet's open_url() to open the created map in a new tab. It's no fully satisfying solution.

Can we somehow render html in flet directly from folium's render_template?
